frontend/src/toolbar/elements/heatmapLogic.ts

```diff
@@ -437,7 +437,11 @@ export const heatmapLogic = kea<heatmapLogicType>([
         scrollDepthPosthogJsError: [
             (s) => [s.posthog],
             (posthog: PostHog): 'version' | 'disabled' | null => {
-                const posthogVersion = posthog?._calculate_event_properties('test', {})?.['$lib_version'] ?? '0.0.0'
+                // Later SDKs have the version in the posthog object, but we need to check for the old way of doing it too
+                const posthogVersion =
+                    posthog.version ??
+                    (posthog as any)?._calculate_event_properties?.('test', {})?.['$lib_version'] ??
+                    '0.0.0'
                 const majorMinorVersion = posthogVersion.split('.')
```

Comment:
Minor point, but we could also replace this with the functions from `semver` (would be along the lines of `compareVersion(a,b) >= 0`

Reply:
I did this here https://github.com/PostHog/posthog/pull/24484